src/components/common/SEO/index.tsx

```diff
@@ -0,0 +1,37 @@
+import Head from 'next/head';
+
+const SEO = () => {
+  const TITLE = 'ACM UCSD Membership Portal';
+  const DESC =
```

Comment:
Updated description (open to suggestions)
`Meet new friends, discover exciting events, and earn points for free ACM swag! Create an account today to find your community!`